### ISSUE TITLE
Stack-2.7.1 error on Mac OS X 10.11

### ISSUE THREAD

**Issue by travelbasys** [commercialhaskell/stack#5539]
Hello. I am on **OS X 10.11.6 El Capitan** .
I was able to install **Stack**, but it refuses to work.
```
$ stack --version
Version 2.7.1, Git revision 8afe0c2932716b0441cf4440d6942c59568b6b19 x86_64 hpack-0.34.4

$ curl -sSL https://get.haskellstack.org/ | sh
Stack version 2.7.1 already appears to be installed at:
  /usr/local/bin/stack

$ stack upgrade
Illegal instruction: 4
```
The same error comes up on 'stack init' and most other commands. Nothing really works, only --help.
WHAT CAN I DO?
Stack could be installed, but refuses to work.
Is my OS too old? Should not the installation process have discovered that?

I am desperately looking for ways to get Haskell to run on my machine but all attempts failed, so far.

Hubert

**Comment by mpilgrem** [comment]
The current version of `stack` is 2.7.5. Are you able to install it and, if you are, do you experience the same problem?

**Comment by travelbasys** [comment]
Dear Readers.

Yes - I new there was a newer stack version at the time, and it worked on my platform.
However, I wanted to study some Haskell App - I forgot its name - which had a component requiring that particular stack version.
It all came to nothing and I abandoned the project.
Thanks, anyway.

Viele Grüße.
Hubert 


Am 13. Mai 2023 23:05:55 MESZ schrieb Mike Pilgrem ***@***.***>:
Closed #5539 as completed.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/commercialhaskell/stack/issues/5539#event-9239203609
